### PR TITLE
Html css popup

### PR DESCRIPTION
**Pull request by maxim-zabolotny**
### html-css-popup

[Demo](https://maxim-zabolotny.github.io/maxim-zabolotny-html-css-popup.github.io/) |
[Code base](https://github.com/maxim-zabolotny/kottans-frontend/tree/main/html-css-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/maxim-zabolotny/HTML-CSS-Popup/index.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+              </div>
+            </div>
+          </div>
+        </li>
+        <li class="user_menu__element">
+          <a href="#"><img src="img/bell.png" class="bell_img" alt="bell"></a>
+        </li>
+        <li class="user_menu__element">
+          <a href="https://github.com/maxim-zabolotny/kottans-frontend"><img class="user_menu__user_photo" src="img/elon musk.jpeg" alt="user_photo"></a>
+        </li>
```
Comment (smillims) [inline submissions/maxim-zabolotny/HTML-CSS-Popup/index.html:117]: Check your website using validator https://validator.w3.org/

File: submissions/maxim-zabolotny/HTML-CSS-Popup/index.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            <img src="img/popup-button.png" alt="apps">
+          </label>
+          <div class="apps_menu">
+            <div class="apps_menu__content">
+              <div class="first_page">
+                <a href="#" class="icon_card">
+                  <img src="img/calendar.png" alt="Calendar">
+                  Calendar
+                </a>
+                <a href="#" class="icon_card">
```
Comment (smillims) [inline submissions/maxim-zabolotny/HTML-CSS-Popup/index.html:31]: I recommend replace your `div` on `ul` or `ol` and wrap your links (each separately) in `li`.
All it for semantics

File: submissions/maxim-zabolotny/HTML-CSS-Popup/index.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+                <label class="more_apps__label" for="m_a_checkbox">More</label>
+                <div class="second_page">
+                  <a href="#" class="icon_card">
+                    <img src="img/chat.png" alt="Chat">
+                    Chat
+                  </a>
+                  <a href="#" class="icon_card">
+                    <img src="img/classroom.png" alt="Classroom">
+                    Classroom
+                  </a>
```
Comment (smillims) [inline submissions/maxim-zabolotny/HTML-CSS-Popup/index.html:87]: Here's the same problem

File: submissions/maxim-zabolotny/HTML-CSS-Popup/style.css [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+    border-radius: 3px;
+  }
+  
+  .icon_card img {
+    max-width: 48px;
+    min-height: 48px;
+    object-fit: contain;
+  }
+  
+
```
Comment (smillims) [inline submissions/maxim-zabolotny/HTML-CSS-Popup/style.css:180]: Add just alone empty line in end of your file.
You can setup your IDE for it work

**Review by smillims — COMMENTED** [review]
Hey, good work! But I found some troubles, let's correct it together

File: submissions/maxim-zabolotny/HTML-CSS-Popup/index.html [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+        </li>
+        <li class="user_menu__element user_menu__element-text">
+          <a href="#">Images</a>
+        </li>
+        <li>
+          <input type="checkbox" class="apps_menu__checkbox" id="apps_checkbox">
+          <label for="apps_checkbox" class="user_menu__element user_menu__element-anchor">
+            <img src="img/popup-button.png" alt="apps">
+          </label>
+          <div class="apps_menu">
```
Comment (lenadgit) [inline submissions/maxim-zabolotny/HTML-CSS-Popup/index.html:24]: popup-menu should have a markup nav > ul > li and be separate of main-menu list.   could you please fix it?

**Review by lenadgit — CHANGES_REQUESTED** [review]

File: submissions/maxim-zabolotny/HTML-CSS-Popup/style.css (hunk 4 above)
Comment (lenadgit) [inline submissions/maxim-zabolotny/HTML-CSS-Popup/style.css:180]: if you see this symbol on github at the end of the file, it means that empty line is needed at the end.
empty line
https://github.com/kottans/frontend-2021-homeworks/pull/232/files
Could you please add an empty line but without any spaces (you added two new lines but with space, Github reads this as a symbol))

**Review by lenadgit — CHANGES_REQUESTED** [review]

File: submissions/maxim-zabolotny/HTML-CSS-Popup/style.css (hunk 4 above)
Comment (maxim-zabolotny) [reply submissions/maxim-zabolotny/HTML-CSS-Popup/style.css]: All fixed
Thanks for your help

File: submissions/maxim-zabolotny/HTML-CSS-Popup/index.html (hunk 1 above)
Comment (maxim-zabolotny) [reply submissions/maxim-zabolotny/HTML-CSS-Popup/index.html]: thanks for the help

File: submissions/maxim-zabolotny/HTML-CSS-Popup/index.html (hunk 2 above)
Comment (maxim-zabolotny) [reply submissions/maxim-zabolotny/HTML-CSS-Popup/index.html]: All fixed
Thanks for your help

File: submissions/maxim-zabolotny/HTML-CSS-Popup/index.html (hunk 3 above)
Comment (maxim-zabolotny) [reply submissions/maxim-zabolotny/HTML-CSS-Popup/index.html]: All fixed
Thanks for your help

File: submissions/maxim-zabolotny/HTML-CSS-Popup/style.css (hunk 4 above)
Comment (maxim-zabolotny) [reply submissions/maxim-zabolotny/HTML-CSS-Popup/style.css]: All fixed
Thanks for your help

**Review by lenadgit — APPROVED** [review]

**Comment by lenadgit** [comment]
Thank you for the submission. Feel free taking the next course module while you submission is being reviewed

**Pull request merged**